### PR TITLE
VP-24-changed button colour in app.css file

### PR DESCRIPTION
**Pull request by RanjiniEswari**

### CODE REVIEW

File: calculator-app/src/App.css [hunk 1]
```diff
@@ -4,39 +4,46 @@
 
 ul.ul-numbers {
   width: 40%;
+  ;
```
Comment (Mutheeswari) [inline calculator-app/src/App.css]: Extra semicolon included

File: calculator-app/src/App.css [hunk 2]
```diff
@@ -4,39 +4,46 @@
...
 .set-width {
   width: 600px;
   height: 100px;
-  font-size: 40px;
-  border-radius: 15px;
+  font-size: 48px;
+  border-radius: 8px;
+  font-family: 'Cutive Mono', monospace;
   text-align: right;
 }
```
Comment (chris-reejoy) [inline calculator-app/src/App.css]: please Do code alignment all over the file

File: calculator-app/src/Buttonnumbers.js [hunk 3]
```diff
@@ -53,14 +53,14 @@ function Buttonnumbers() {
             className="button-cldel"
             onClick={() => setDigit(0)}
           >
-            Clear
+            CLEAR
```
Comment (prabhugberlin) [inline calculator-app/src/Buttonnumbers.js]: use constants variable for hard coded text

Comment (RanjiniEswari) [reply calculator-app/src/Buttonnumbers.js]: As per your comments CLEAR and DEL are included in  "constants.js" file.

**Review by Mutheeswari — APPROVED** [review]

File: calculator-app/src/Buttonnumbers.js (hunk 3 above)
Comment (prabhugberlin) [reply calculator-app/src/Buttonnumbers.js]: use object like below in constant.js
export const button =  {
  'clear': 'CLEAR',
  'delete': 'DEL'
}

Comment (RanjiniEswari) [reply calculator-app/src/Buttonnumbers.js]: As per your comments used object in constants.js

**Review by prabhugberlin — APPROVED** [review]

**Review by chris-reejoy — APPROVED** [review]

**Pull request merged**